src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/OnesComplementInstruction.tt

```diff
@@ -0,0 +1,10 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".cs" #>
+<#
+string instruction = "OnesComplement";
```

Comment:
The only thing about this class that stops it being a perfect subset of NotInstruction is that it reports "OnesComplement" as it's name. Would just removing this class confuse debugging too much?